source/going_further.rst

```diff
@@ -0,0 +1,478 @@
+#############
+Going Further
+#############
+
+This section shows some helpful tools that might aid you along your programming path. We briefly describe here how
+to structure larger projects and work with multiple source files. We discuss python coding style guide (also known as PEP8).
+and how pylint can help you to write and maintain readable and stylish code. At the end there is a list of resources you might find
+helpful in your day-to-day python programming.
+
+
+Managing larger projects
+========================
+
+Divide and conquer
+--------------------
+
+Up to this point, you might have put all of your source coed in one file. For small libraries and project this is fine, but when complexity
+rises it is better to divide your code into smaller files, that is to create libraries or modules. It is not necessary, for example, to put
+every class in a separate file, but having one file for classes that are logically connected is better for readability.
+Going through a file that has thousands lines of code can be difficult.
```

Comment:
Actually, it sounds weird to me. It's like "it's better to divide. It's not necessary to devide, but you can keep it all in one file. Large files are bad." It's somehow about the order of this sentences, and the "but" word. I would change it into sth like:
"...to create libraries or modules. Going through a file that has thousands lines of code can be difficult. However it is not necessary to, for example, put every class in a separate file; having one file for classes (...) is better for readability."
But maybe I'm just nitpicking ;)